test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1848,7 +1849,9 @@ class test_ucp_am_nbx_rndv_memtype : public test_ucp_am_nbx_rndv {
 
 UCS_TEST_P(test_ucp_am_nbx_rndv_memtype, rndv)
 {
-    test_am_send_recv_memtype(64 * UCS_KBYTE);
+    ucp_context_config_t *cfg = &sender().worker()->context->config.ext;
+    size_t size               = cfg->rndv_frag_size[UCS_MEMORY_TYPE_HOST] + 1;
```

Comment:
for my understanding: part of the test is that we make `size` too big for lane/rkey to be used?

Reply:
i just make sure that we are using more than 1 fragment when pipeline protocol is used (should be always selected for large cuda-managed transfers)